compose/material/material/src/desktopMain/kotlin/androidx/compose/material/DesktopMenu.desktop.kt

```diff
@@ -115,12 +123,31 @@ fun DropdownMenu(
                 }
             },
         ) {
+            val focusManager = LocalFocusManager.current
+
             DropdownMenuContent(
                 expandedStates = expandedStates,
                 transformOriginState = transformOriginState,
-                modifier = modifier,
+                modifier = modifier.onKeyEvent {
+                    if (it.type != KeyEventType.KeyDown) return@onKeyEvent false
+                    when (it.key) {
+                        Key.DirectionDown -> {
+                            focusManager.moveFocus(FocusDirection.Next)
+                            true
+                        }
+                        Key.DirectionUp -> {
+                            focusManager.moveFocus(FocusDirection.Previous)
+                            true
+                        }
+                        else -> false
+                    }
+                },
                 content = content
             )
+
+            LaunchedEffect(Unit) {
+                focusManager.moveFocus(FocusDirection.Next)
```

Comment:
We probably shouldn't move the focus on the first item. Native UI don't do it.

We can keep the focus on the root container after we show a popup.